### ISSUE TITLE
Cant Save Profile without Password

### ISSUE THREAD

**Issue by velkymx** [ProcessMaker/spark#1612]
I should be able to save my profile without changing my password. It currently requires I change my password even when I don't want to. Validation should happen only if the first password box contains something.

![Screenshot 2019-04-01 09 43 30](https://user-images.githubusercontent.com/535300/55344666-cd55ef00-5462-11e9-9330-eb257024d90c.png)

**Comment by velkymx** [comment]
Also the password fields should not prepopulate (as in the screenshot above)

**Comment by velkymx** [comment]
It's also changing my form values after the page loads. 

<img width="309" alt="Screenshot 2019-04-03 10 13 07" src="https://user-images.githubusercontent.com/535300/55498732-31122080-55f9-11e9-928f-e0892ac0ca8c.png">

**Comment by PMQABen** [comment]
I tested the problem on what we have up on "develop.bpm4.qa", and I could not reproduce it. I created a separate user, logged in as them, and every time I saved a change to any part of that user's profile, it never prompted me to enter the password. The only time I was prompted to enter the password is if I intentionally went to update the new password field.

I know there was code committed for this on #1628, but I'm not certain this is still an issue.
